taskt/Core/Automation/Commands/Variable Commands/NewVariableCommand.cs

```diff
@@ -3,69 +3,64 @@
 using System.Linq;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
-
-
     [Serializable]
-    [Attributes.ClassAttributes.Group("Variable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to explicitly add a variable if you are not using **Set Variable* with the setting **Create Missing Variables** at runtime.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to modify the value of variables.  You can even use variables to modify other variables.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements actions against VariableList from the scripting engine.")]
+    [Group("Variable Commands")]
+    [Description("This command explicitly adds a new variable or can update the one if already exists.")]
```

Comment:
This is a bit awkwardly worded. Lets go with "updates an existing variable."

Reply:
Updated